openlibrary/solr/update_work.py

```diff
@@ -311,6 +310,32 @@ def datetimestr_to_int(datestr):
     return int(time.mktime(t.timetuple()))
 
 
+class EbookAccess(OrderedEnum):
+    # Keep in sync with solr/conf/enumsConfig.xml !
+    NO_EBOOK = 0
+    UNCLASSIFIED = 1
+    PRINTDISABLED = 2
+    BORROWABLE = 3
+    PUBLIC = 4
+
+    def to_solr_str(self):
+        return self.name.lower()
+
+
+def get_ia_access_enum(
+    collections: set[str],
+    access_restricted_item: bool,
+) -> EbookAccess:
+    if 'inlibrary' in collections:
+        return EbookAccess.BORROWABLE
+    elif 'printdisabled' in collections:
+        return EbookAccess.PRINTDISABLED
+    elif access_restricted_item or not collections:
+        return EbookAccess.UNCLASSIFIED
```

Comment:
In these cases, the `if` statement is telling us these are `restricted` items. I think that's a better fit than `unclassified` see also notes above.